src/plugins/stake-plugins/generic/util/stakeKitUtils.ts

```diff
@@ -63,3 +71,10 @@ export const yieldGetSingleYieldBalances = async (integrationId: string, yieldBa
 export const yieldYieldOpportunity = async (integrationId: string): Promise<YieldDto> => {
   return await fetchGet<YieldDto>(`/v1/yields/${integrationId}`)
 }
+
+const checkInsufficientFunds = (res: any): void => {
```

Comment:
I generally prefer `unknown` over `any`, since it prevents you from messing with the value until you clean it. Much safer.